cmd/cli.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/did"
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+
+	app := &cli.App{
```

Comment:
mostly thinking how does this scale to many commands? maybe a more modular structure

Reply:
yea....i was thinking we could refactor once we actually start to scale out features. Was a little concerned about causing unnecessary redirect when it's currently so simple. What do you think? modularize it now or later? Either way, at some point: agreed that needs to happen.